### PR TITLE
Adding benchmarks for BoltDB

### PR DESCRIPTION
**Pull request by deepakjois**
@mrjn This is just informative, in case you (or somebody else) are interested in looking at the benchmark code.

Will merge immediately.

### CODE REVIEW

File: bench_test.go [hunk 1]
```diff
@@ -189,6 +198,30 @@ func BenchmarkReadRandomLmdb(b *testing.B) {
...
+			boltBkt := txn.Bucket([]byte("bench"))
+			y.AssertTrue(boltBkt != nil)
+			for pb.Next() {
+				key := newKey()
+				v := boltBkt.Get(key)
+				if v == nil {
+					c.notFound++
+					continue
+				}
+				y.AssertTruef(len(v) == *flagValueSize, "Assertion failed. value size is %d, expected %d", len(v), *flagValueSize)
```
Comment (manishrjain) [inline bench_test.go]: Avoid the `f` suffixes on Asserts unless you absolutely need them. They're much slower.

File: populate/main.go [hunk 2]
```diff
@@ -88,6 +90,21 @@ func writeBatch(entries []*badger.Entry) int {
 
 	}
 
+	if boltdb != nil {
+		err := boltdb.Update(func(txn *bolt.Tx) error {
+			boltBkt := txn.Bucket([]byte("bench"))
+			y.AssertTrue(boltBkt != nil)
+			for _, e := range entries {
+				err := boltBkt.Put(e.Key, e.Value)
```
Comment (manishrjain) [inline populate/main.go]: In Go:

`if err := statement(); err != nil { handle }`

Comment (deepakjois) [inline populate/main.go]: Done

File: bench_test.go (hunk 1 above)
Comment (deepakjois) [inline bench_test.go]: Even when the asserts are false?

**Pull request merged**